getting_started_scenarios/ssm_scenario/SPECIFICATION.md

```diff
@@ -0,0 +1,122 @@
+Upon completion of the program, all AWS resources are cleaned up.
+Let's get started...
+Please hit Enter
+
+
+--------------------------------------------------------------------------------
+Create an SSM maintenance window.
+Please enter the maintenance window name (default is ssm-maintenance-window):
+
+The maintenance window id is mw-0a782f69416fa2d68
+--------------------------------------------------------------------------------
+Modify the maintenance window by changing the schedule
+Please hit Enter
+
+The SSM maintenance window was successfully updated
+--------------------------------------------------------------------------------
+Create an SSM document that defines the actions that Systems Manager performs on your managed nodes.
+Please enter the document name (default is ssmdocument):
+
+The status of the SSM document is Creating
+--------------------------------------------------------------------------------
+Now we are going to run a command on an EC2 instance
+Please hit Enter
+
+The SSM document is active and ready to use.
+Command ID: ac4aef03-7d60-4fba-8d71-051defea89e2
+Wait 5 secs
+Command execution successful
+--------------------------------------------------------------------------------
+Lets get the time when the specific command was sent to the specific managed node
+Please hit Enter
+
+The time of the command invocation is 2024-04-09 14:52:50
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+ Now we will create an SSM OpsItem.
+ SSM OpsItem is a feature provided by the Systems Manager service.
+ It is a type of operational data item that allows you to manage and track various operational issues,
+ events, or tasks within your AWS environment.
+
+ You can create OpsItems to track and manage operational issues as they arise.
+ For example, you could create an OpsItem whenever your application detects a critical error
+ or an anomaly in your infrastructure.
+
+Please hit Enter
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+Now we will update SSM OpsItem oi-c1e5435471e9
+Please hit Enter
+
+Now we will resolve the SSM OpsItem oi-c1e5435471e9
+Please hit Enter
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+Would you like to delete the AWS Systems Manager resources? (y/n)
+
+The AWS Systems Manager resources will not be deleted
+--------------------------------------------------------------------------------
+This concludes the AWS Systems Manager SDK Getting Started scenario.
+--------------------------------------------------------------------------------
+
+   ```
+## SOS Tags
```

Comment:
SOS is an internal name and should be used in these public docs.

Reply:
fixed